src/ProjectTemplates/Web.Spa.ProjectTemplates/content/React-CSharp/Startup.cs

```diff
@@ -87,17 +101,20 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 #endif
             app.UseStaticFiles();
             app.UseSpaStaticFiles();
-#if (IndividualLocalAuth)
 
+            app.UseRouting();
+
+#if (IndividualLocalAuth)
             app.UseAuthentication();
             app.UseIdentityServer();
 #endif
+            app.UseAuthorization();
```

Comment:
I think `UseAuthorization` should only be included when authentication is enabled.